packages/sanity/src/structure/panes/document/DocumentPaneProvider.tsx

```diff
@@ -533,7 +535,8 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
       isLocked ||
       isDeleting ||
       isDeleted ||
-      isLiveEditAndDraft
+      isLiveEditAndDraft ||
+      isCreateLinked
```

Comment:
Not clear from the diff here, but this sets the documentPane `readOnly` property.

Note: Not all codepaths respect document path `readOnly`state. For instance, `Paste document` can mutate a document pane in `readOnly` mode. I contend that this is the right place to put this `readOnly` code for Create though; code that does not respect it should be corrected elsewhere.